test/integration/file-uploads/request-body.test.js

```diff
@@ -645,4 +645,81 @@ describe('file upload in request body', function () {
                 .to.equal('Binary file load error: file resolver interface mismatch');
         });
     });
+
+    (IS_BROWSER ? describe.skip : describe)('large file upload in request body', function () {
+        // eslint-disable-next-line mocha/no-sibling-hooks
+        before(function (done) {
+            this.run({
+                fileResolver: {
+                    stat: function (src, cb) {
+                        cb(null, {isFile: function () { return true; }, mode: 33188});
+                    },
+                    createReadStream: function () {
```

Comment:
Why it's returning a buffer instead of a[.](https://gist.github.com/wpscholar/270005d42b860b1c33cf5ab25b37928a) stream?

Reply:
Because you can't re-use the same stream. Return a new stream here instead of pushing data to a common stream.